module/state_synchronization/execution_data.go

```diff
@@ -9,7 +9,7 @@ import (
 type ExecutionData struct {
 	BlockID            flow.Identifier
 	Collections        []*flow.Collection
-	Events             []*flow.Event
+	Events             []flow.EventsList
```

Comment:
we're updating this to a 2d list, is that intentional? does it affect the ingestion side from DPS?

Reply:
Yes, previously the events for a block were flattened, here we've grouped them by Chunk (i believe?). If anything, I'd assume it would make ingestion simpler for DPS since the events would be pre-grouped.